core/include/usModuleInfo.h

```diff
@@ -48,6 +49,7 @@ struct US_Core_EXPORT ModuleInfo
   std::string name;
   std::string location;
   std::string autoLoadDir;
+  std::map<std::string, std::string> properties;
```

Comment:
I believe the properties member is not needed here and the module can access the framework launch properties via the core context. Actually, the complete class can probably be remove since it was a helper construct during static initialization and registration of a module. But now we have explicit install calls.